packages/frameworks/frameworks.json

```diff
@@ -682,6 +682,33 @@
       }
     }
   },
+  {
+    "name": "RedwoodJS",
+    "slug": "redwoodjs",
+    "logo": "https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/redwood.svg",
+    "tagline": "RedwoodJS is a full-stack framework for the Jamstack.",
+    "description": "A RedwoodJS app, bootstraped with create-redwood-app.",
+    "website": "https://redwoodjs.com",
+    "detectors": {
+      "every": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@redwoodjs\\/core\":\\s*\".+?\"[^}]*}"
+        }
+      ]
+    },
+    "settings": {
+      "buildCommand": {
+        "value": "yarn rw db up --no-db-client --auto-approve && yarn rw build"
+      },
+      "devCommand": {
+        "value": "yarn rw dev"
```

Comment:
Don't we need to add `--port $PORT` or something to pass the port?

Reply:
They don't support that flag unfortunately, they use a [redwood.toml](https://redwoodjs.com/docs/app-configuration-redwood-toml) file to set the port.

Reply:
My understanding is that this dev command won't be used, since the `@vercel/redwood` Runtime is special-cased. Is that correct?

Reply:
It uses `config.devCommand` here:
https://github.com/vercel/vercel/blob/041e8cc601befc5e9e745daf7c098fed4914f889/packages/redwood/src/index.ts#L92

So it could be overridden if the user changes their project settings.